app/admin/quests/create/page.tsx

```diff
@@ -0,0 +1,1049 @@
+              <div className="flex gap-2 items-center">
+                <p>Boost this quest</p>
+                <Switch
+                  name="Boost this Quest"
+                  value={showBoost}
+                  onChange={() => setShowBoost((prev) => !prev)}
+                />
+              </div>
+            </div>
+            {showBoost ? (
+              <div className="flex flex-col w-full gap-8">
+                <Textinput
+                  onChange={handleBoostInputChange}
+                  value={boostInput.num_of_winners}
+                  name="num_of_winners"
+                  label="Number of winners"
+                  placeholder="Number of winners"
+                />
+                <Dropdown
+                  value={boostInput.token}
+                  backgroundColor="#29282B"
+                  textColor="#fff"
+                  // fullWidth
+                  // sx={{
+                  //   backgroundColor: "#1F1F25",
+                  //   borderRadius: "8px",
+                  // }}
+                  // value={boostInput.token}
+                  // label="Token"
+                  handleChange={(event: SelectChangeEvent) => {
+                    setBoostInput((prev) => ({
+                      ...prev,
+                      token: event.target.value,
+                      token_decimals:
+                        TOKEN_DECIMAL_MAP[
+                          event.target.value as keyof typeof TOKEN_DECIMAL_MAP
+                        ],
+                    }));
+                  }}
+                  options={Object.keys(TOKEN_ADDRESS_MAP[network]).map(
+                    (eachItem) => {
+                      return {
+                        value: eachItem,
+                        label: eachItem,
+                      };
+                    }
+                  )}
+                />
+                <Textinput
+                  onChange={handleBoostInputChange}
+                  value={boostInput.amount}
+                  name="amount"
+                  label="Amount"
+                  placeholder="Amount"
+                />
+              </div>
+            ) : null}
+          </InputCard>
+
+          <div className="w-full items-center justify-center flex">
+            <div className="w-fit">
+              <Button
+                loading={buttonLoading}
+                onClick={async () => {
+                  const id = await handleCreateQuest();
+                  if (!id) return;
+                  await handleCreateBoost(id);
+                  handlePagination("Next");
+                }}
+                disabled={isButtonDisabled}
+              >
+                <p>Confirm Next</p>
+              </Button>
+            </div>
+          </div>
+          <QuizControls
+            step={currentPage}
+            setStep={setCurrentPage}
+            onCancel={() => router.push("/admin/quests")}
+          />
+        </>
+      ) : currentPage === 3 ? (
```

Comment:
As GPT proposed you can make divide each page into a component and even go deeper and ask GPT to build the component for you (works pretty well)

I'll let you manage :)